### DISCUSSION TITLE
Arquivo csproj nao aparecer no gerenciador do visual studio

### DISCUSSION THREAD

**Discussion by syspone** [balta-io/forum#152] [Tira Dúvidas] [answered]
Professor, @andrebaltieri !

Apesar do csproj está dentro da pasta do projeto, porem no gerenciador do visual studio nao aparece. Possui algum procedimento pra ele aparecer?

![image](https://user-images.githubusercontent.com/48097255/206627370-19e2ea6c-a35a-465e-9beb-d60304a2e13b.png)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @cmsoouza, como vai?
Ele não aparece mesmo... na verdade ele está aparecendo, mas não fica igual no VSCode...

Você pode editar ele pelo VS Code, ou clicar sobre o **CursoCsharpx** (Segundo item de cima para baixo na sua solution) com o botão direito e ir em EDIT.

**Reply by syspone** [reply]
vlw professor @andrebaltieri  deu certo!
